### DISCUSSION TITLE
edelweiss 0.8.0

### DISCUSSION THREAD

**Discussion by jmitrevs** [fastmachinelearning/hls4ml#926] [Announcements]
## What's Changed
* Decouple pipeline style from strategy by @vloncar in https://github.com/fastmachinelearning/hls4ml/pull/781
* Don't use reader in ModelGraph and layers by @vloncar in https://github.com/fastmachinelearning/hls4ml/pull/770
* Remove tf_to_hls by @vloncar in https://github.com/fastmachinelearning/hls4ml/pull/795
* [pre-commit.ci] pre-commit autoupdate by @pre-commit-ci in https://github.com/fastmachinelearning/hls4ml/pull/796
* Fix parsing of QConv2DBatchnorm weights by @vloncar in https://github.com/fastmachinelearning/hls4ml/pull/802
* [pre-commit.ci] pre-commit autoupdate by @pre-commit-ci in https://github.com/fastmachinelearning/hls4ml/pull/801
* Discussion - Inlined Conv slows down latency significantly (up to x15 - x20) by @bo3z in https://github.com/fastmachinelearning/hls4ml/pull/800
* [pre-commit.ci] pre-commit autoupdate by @pre-commit-ci in https://github.com/fastmachinelearning/hls4ml/pull/807
* Fix over-allocation of bits for quantised po2 by @bo3z in https://github.com/fastmachinelearning/hls4ml/pull/806
* Propagate zeros from Conv layers to multiplication config by @bo3z in https://github.com/fastmachinelearning/hls4ml/pull/797
* Fix Vitis Conv1D/2D latency strategy by @vloncar in https://github.com/fastmachinelearning/hls4ml/pull/815
* Improved parsing of pytorch models using torch.FX - Clean by @JanFSchulte in https://github.com/fastmachinelearning/hls4ml/pull/799
* [pre-commit.ci] pre-commit autoupdate by @pre-commit-ci in https://github.com/fastmachinelearning/hls4ml/pull/816
* Support for parsing nested models by @vloncar in https://github.com/fastmachinelearning/hls4ml/pull/794
* Fix loading weights in n-dim dense -> 1x1 conv by @vloncar in https://github.com/fastmachinelearning/hls4ml/pull/821
* [pre-commit.ci] pre-commit autoupdate by @pre-commit-ci in https://github.com/fastmachinelearning/hls4ml/pull/828
* Fix loading weights in GarNetStacked and GarNet internal array precisions by @joshlerner in https://github.com/fastmachinelearning/hls4ml/pull/827
* [pre-commit.ci] pre-commit autoupdate by @pre-commit-ci in https://github.com/fastmachinelearning/hls4ml/pull/830
* Fix profiling for GRU/LSTM by @drankincms in https://github.com/fastmachinelearning/hls4ml/pull/833
* [pre-commit.ci] pre-commit autoupdate by @pre-commit-ci in https://github.com/fastmachinelearning/hls4ml/pull/835
* remove obsolete and unused docker directory by @jmitrevs in https://github.com/fastmachinelearning/hls4ml/pull/836
* [pre-commit.ci] pre-commit autoupdate by @pre-commit-ci in https://github.com/fastmachinelearning/hls4ml/pull/842
* Remove obsolete parameter mapping between pytorch and keras by @JanFSchulte in https://github.com/fastmachinelearning/hls4ml/pull/847
* Make binary CNN match between Keras and hls4ml by @jmitrevs in https://github.com/fastmachinelearning/hls4ml/pull/804
* No longer make ExponentPrecisionType and XnorPrecisionType inherit from IntegerPrecisionType by @jmitrevs in https://github.com/fastmachinelearning/hls4ml/pull/845
* Add support for flattening to the pytorch parser by @JanFSchulte in https://github.com/fastmachinelearning/hls4ml/pull/852
* Add option to configure IP version by @AdrianAlan in https://github.com/fastmachinelearning/hls4ml/pull/851
* Bug fix for named nn.Sequential in pytorch parser by @JanFSchulte in https://github.com/fastmachinelearning/hls4ml/pull/848
* Add QDepthwiseConv2D, DepthwiseConv2D, DepthwiseConv1D support by @jmitrevs in https://github.com/fastmachinelearning/hls4ml/pull/834
* Symbolic expressions in hls4ml by @vloncar in https://github.com/fastmachinelearning/hls4ml/pull/660
* Update dependencies, add testing extras by @jmitrevs in https://github.com/fastmachinelearning/hls4ml/pull/837
* Bump actions/checkout from 3 to 4 by @dependabot in https://github.com/fastmachinelearning/hls4ml/pull/866
* [pre-commit.ci] pre-commit autoupdate by @pre-commit-ci in https://github.com/fastmachinelearning/hls4ml/pull/869
* try to use new runners for gitlab CI by @jmitrevs in https://github.com/fastmachinelearning/hls4ml/pull/879
* [pre-commit.ci] pre-commit autoupdate by @pre-commit-ci in https://github.com/fastmachinelearning/hls4ml/pull/880
* Fix weight precision format string by @vloncar in https://github.com/fastmachinelearning/hls4ml/pull/877
* add acknowledgments by @jmduarte in https://github.com/fastmachinelearning/hls4ml/pull/862
* Support for quantized SeparableConv1D/2D by @vloncar in https://github.com/fastmachinelearning/hls4ml/pull/861
* Speed up Keras profiling by @AdrianAlan in https://github.com/fastmachinelearning/hls4ml/pull/863
* [pre-commit.ci] pre-commit autoupdate by @pre-commit-ci in https://github.com/fastmachinelearning/hls4ml/pull/882
* Fix profiling SeparableConv1D and SeparableConv2D by @qberthet in https://github.com/fastmachinelearning/hls4ml/pull/891
* Add support for filt_height==1 for streaming quartus conv2d by @jmitrevs in https://github.com/fastmachinelearning/hls4ml/pull/886
* Fix config structure name in pragma for SeparableConv1D by @qberthet in https://github.com/fastmachinelearning/hls4ml/pull/884
* [pre-commit.ci] pre-commit autoupdate by @pre-commit-ci in https://github.com/fastmachinelearning/hls4ml/pull/895
* Fix bit overflow with softmax by @calad0i in https://github.com/fastmachinelearning/hls4ml/pull/887
* bump 0.8.0rc1 by @jmitrevs in https://github.com/fastmachinelearning/hls4ml/pull/915
* [pre-commit.ci] pre-commit autoupdate by @pre-commit-ci in https://github.com/fastmachinelearning/hls4ml/pull/902
* Add funding acknowledgements by @jmduarte in https://github.com/fastmachinelearning/hls4ml/pull/918
* Fix fetching models from example-models repo by @vloncar in https://github.com/fastmachinelearning/hls4ml/pull/919
* add blank line to make rst format correct by @jmitrevs in https://github.com/fastmachinelearning/hls4ml/pull/923
* Update default FPGA part number from KU115 to VU13P by @jmduarte in https://github.com/fastmachinelearning/hls4ml/pull/924
* update to 0.8.0 by @jmitrevs in https://github.com/fastmachinelearning/hls4ml/pull/925

## New Contributors
* @pre-commit-ci made their first contribution in https://github.com/fastmachinelearning/hls4ml/pull/796
* @joshlerner made their first contribution in https://github.com/fastmachinelearning/hls4ml/pull/827
* @qberthet made their first contribution in https://github.com/fastmachinelearning/hls4ml/pull/891

**Full Changelog**: https://github.com/fastmachinelearning/hls4ml/compare/v0.7.1...v0.8.0

<hr /><em>This discussion was created from the release <a href='https://github.com/fastmachinelearning/hls4ml/releases/tag/v0.8.0'>edelweiss 0.8.0</a>.</em>
